Example/FinancialConnections Example/FinancialConnections Example/WebViewViewController.swift

```diff
@@ -37,11 +37,16 @@ import WebKit
 // 2. Add client code (shown below) that will open an extra browser that
 //    will handle the bank OAuth pop-up.
 //
-// 3. When Financial Connections calls the `return_url`, your website code
+// 3. Optional, but recommended: Handle app-to-app authentication flows by
```

Comment:
@mats-stripe let's make this non-optional to ensure merchants get the best experience
```suggestion
// 3. Handle app-to-app authentication flows by
```